procdesc.cc

```diff
@@ -542,15 +543,25 @@ FORK_TEST(Pdfork, OtherUserIfRoot) {
   EXPECT_EQ(EPERM, errno);
   EXPECT_PID_ALIVE(pid);
 
+#ifdef __FreeBSD__
+  // Can't send a signal to a process with another UID unless we are root.
```

Comment:
Isn't this test case always going to run as root because of line 523?

Reply:
By the time this runs we have changed UID to the unprivileged user `EXPECT_EQ(0, setuid(other_uid));` in line 536.

Reply:
Ah, I missed that (didn't expand the skipped bits in the GitHub diff).